### PR TITLE
exploitdb 2020-10-30

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/exploitdb.rb [hunk 1]
```diff
@@ -2,9 +2,9 @@ class Exploitdb < Formula
   desc "Database of public exploits and corresponding vulnerable software"
   homepage "https://www.exploit-db.com/"
   url "https://github.com/offensive-security/exploitdb.git",
-      tag:      "2020-10-28",
-      revision: "17bbfdaf380c10065312e056c117667a746cff1e"
-  version "2020-10-28"
+      tag:      "2121-10-30",
```
Comment (issyl0) [inline Formula/exploitdb.rb:5]:
```suggestion
      tag:      "2020-10-30",
```

File: Formula/exploitdb.rb [hunk 2]
```diff
@@ -2,9 +2,9 @@ class Exploitdb < Formula
...
+      revision: "48bd7b3ea62d71d3bab2c9b428e4ac40a656ee4f"
+  version "2121-10-30"
```
Comment (issyl0) [inline Formula/exploitdb.rb:7]:
```suggestion
  version "2020-10-30"
```

**Review by issyl0 — DISMISSED** [review]
As much as we might all want 2020 to be over, it isn't yet. 😆

File: Formula/exploitdb.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/exploitdb.rb:5]: haha, yeah quite a fast move :)

**Review by zachauten — APPROVED** [review]

**Review by dtrodrigues — APPROVED** [review]

**Pull request merged**